### PR TITLE
ECOM-4904 remove the program editor from Programs IDA

### PR DESCRIPTION
**Pull request by schenedx**
@rlucioni @mikedikan Please review

This is a removal of all frontend assets from the Program IDA

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,18 +5,13 @@ sudo: false
...
 before_install:
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
 install:
   - pip install -U pip wheel codecov
   - pip install -r requirements/test.txt
-  - make requirements.js
 script:
-    # Compile assets and run validation
   - make static -e DJANGO_SETTINGS_MODULE="programs.settings.test"
```
Comment (rlucioni) [inline .travis.yml]: You can remove this.

File: Makefile [hunk 2]
```diff
@@ -22,52 +21,38 @@ help:
...
-
-requirements: requirements.js
+requirements:
 	pip install -qr requirements/local.txt --exists-action w
 
 test: clean
 	REUSE_DB=1 coverage run ./manage.py test programs --settings=programs.settings.test
 	coverage report
 
+static:
```
Comment (rlucioni) [inline Makefile]: I think this entire target can go. There are no more static assets to collect.

**Comment by rlucioni** [comment]
@schenedx looking good. Could you please also remove CORS-related dependencies and config? Since the authoring app now lives in Studio, this support is no longer necessary. Take a look at the PR where I introduced it to figure out what to remove: https://github.com/edx/programs/pull/51.

File: Makefile [hunk 3]
```diff
@@ -22,52 +21,38 @@ help:
 	@echo "  requirements               install requirements for local development"
 	@echo "  requirements.js            install JavaScript requirements for local development"
```
Comment (mikedikan) [inline Makefile]: Remove requirements.js from the help text

**Comment by rlucioni** [comment]
👍 once comments addressed.

**Comment by rlucioni** [comment]
These changes have also shaved more than a minute off of the project build time! 🎆

**Comment by schenedx** [comment]
@rlucioni Well, we are putting more unit tests and sass into studio, which is already too big 😞

**Comment by rlucioni** [comment]
@schenedx true, though to be fair Studio already installs a lot of the assets necessary for the app via npm and bower. I think those two took up most of the build time before.

**Comment by rlucioni** [comment]
@schenedx speaking of which, you can also get rid of `.nvmrc` since this project no longer requires node.

**Comment by rlucioni** [comment]
Before merging this, please update https://github.com/edx/configuration/blob/master/playbooks/roles/programs/tasks/main.yml so that CI builds and deployments continue to function.

**Comment by schenedx** [comment]
@rlucioni Thank you for the reminder. The configuration repo changes are made at https://github.com/edx/configuration/pull/3262

**Comment by rlucioni** [comment]
🎉

**Comment by mikedikan** [comment]
👍

**Pull request merged**